isomdurm/day2.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+  <link rel="stylesheet" href="day2.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+  <title>Isom Durm</title>
+  <link rel="shortcut icon" type="image/x-icon" href="file:///Users/forpatientuse/Desktop/1443595223.ico">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="col-xs-5">
+        
+        <h1><img src="http://33.media.tumblr.com/tumblr_m5u04lKMQj1rqlfp9.gif"></img><br><br>Questions</h1>
+          <ul>
+           <li>I have been having conflicting difficulties between Ruby standards and Javascript standards. I have not learned enough about either to fully grasp the use of the language, but more so about JS. Is there a way I can try to get off of the Ruby Bandwagon to reroute my education process?</li><br>
+           <li>Throughout these lessons for this homework, I am also having difficulty grasping the white board method for code. Once I see it now, I know exactly what it means and does, such as the if, then, for, while, etc., but I do not feel comfortable writing it blankly. This coincides with the previous question. There's a clear element I am not grabbing by the horns.</li><br>
+           <li>I would love loVE LOVE to start using the Ternary Operators instead of the other method, but again, I am having trouble transitioning from each. How can we solve this?</li>
```

Comment:
you really should use if else (not ternary) for almost everything.  Only use ternary for super super simple things.